src/uct/sm/mm/mm_posix.c

```diff
@@ -17,12 +17,16 @@
 #define UCT_MM_POSIX_MMAP_PROT      (PROT_READ | PROT_WRITE)
 #define UCT_MM_POSIX_HUGETLB        UCS_BIT(0)
 #define UCT_MM_POSIX_SHM_OPEN       UCS_BIT(1)
-#define UCT_MM_POSIX_CTRL_BITS      2
+#define UCT_MM_POSIX_PROC_LINK      UCS_BIT(2)
+#define UCT_MM_POSIX_CTRL_BITS      3
+#define UCT_MM_POSIX_FD_BITS        29
```

Comment:
@alinask fd and proc id should be 32 bits each, and we need 3 bits for ctrl bits, but they cannot be hold with in 64 bits of mm_id. Currently I do not have a good solution for this.